runme.py

```diff
@@ -7,160 +7,18 @@
-    return optim
-
-
-def run_scenario(input_file, timesteps, scenario, result_dir, dt,
-                 plot_tuples=None,  plot_sites_name=None, plot_periods=None,
-                 report_tuples=None, report_sites_name=None):
-    """ run an urbs model for given input, time steps and scenario
-
-    Args:
-        input_file: filename to an Excel spreadsheet for urbs.read_excel
-        timesteps: a list of timesteps, e.g. range(0,8761)
-        scenario: a scenario function that modifies the input data dict
-        result_dir: directory name for result spreadsheet and plots
-        dt: length of each time step (unit: hours)
-        plot_tuples: (optional) list of plot tuples (c.f. urbs.result_figures)
-        plot_sites_name: (optional) dict of names for sites in plot_tuples
-        plot_periods: (optional) dict of plot periods(c.f. urbs.result_figures)
-        report_tuples: (optional) list of (sit, com) tuples (c.f. urbs.report)
-        report_sites_name: (optional) dict of names for sites in report_tuples
-
-    Returns:
-        the urbs model instance
-    """
-
-    # scenario name, read and modify data for scenario
-    sce = scenario.__name__
-    data = urbs.read_excel(input_file)
-    data = scenario(data)
-    urbs.validate_input(data)
-
-    # create model
-    prob = urbs.create_model(data, dt, timesteps)
-
-    # refresh time stamp string and create filename for logfile
-    now = prob.created
-    log_filename = os.path.join(result_dir, '{}.log').format(sce)
-
-    # solve model and read results
-    optim = SolverFactory('glpk')  # cplex, glpk, gurobi, ...
-    optim = setup_solver(optim, logfile=log_filename)
-    result = optim.solve(prob, tee=True)
-
-    # save problem solution (and input data) to HDF5 file
-    urbs.save(prob, os.path.join(result_dir, '{}.h5'.format(sce)))
-
-    # write report to spreadsheet
-    urbs.report(
-        prob,
-        os.path.join(result_dir, '{}.xlsx').format(sce),
-        report_tuples=report_tuples,
-        report_sites_name=report_sites_name)
-
-    # result plots
-    urbs.result_figures(
-        prob,
-        os.path.join(result_dir, '{}'.format(sce)),
-        timesteps,
-        plot_title_prefix=sce.replace('_', ' '),
-        plot_tuples=plot_tuples,
-        plot_sites_name=plot_sites_name,
-        periods=plot_periods,
-        figure_size=(24, 9))
-    return prob
-
-
 if __name__ == '__main__':
     input_file = 'mimo-example.xlsx'
     result_name = os.path.splitext(input_file)[0]  # cut away file extension
-    result_dir = prepare_result_directory(result_name)  # name + time stamp
+    result_dir = urbs.prepare_result_directory(result_name)  # name+time stamp
 
     # copy input file to result directory
     shutil.copyfile(input_file, os.path.join(result_dir, input_file))
     # copy runme.py to result directory
     shutil.copy(__file__, result_dir)
+    
+    # Choose solver (cplex, glpk, gurobi, ...)
+    Solver = 'gurobi'
```

Comment:
why is the variable starting with a capital letter?

Reply:
No reason for that. I can change that.